### ISSUE TITLE
One-way collisions

### ISSUE THREAD

**Issue by elvisish** [mapeditor/tiled-to-godot-export#55]
Are one-way collisions still planned?

**Comment by cromerc** [comment]
It has been on the "todo" list in the README for 3 years... I wouldn't expect it any time soon.

**Comment by cromerc** [comment]
#51

**Comment by elvisish** [comment]
> #51 

I'm guessing not.

**Comment by bjorn** [comment]
There's just nobody who found time to finish the change. Maybe you can help by giving feedback on that change?

**Comment by elvisish** [comment]
> There's just nobody who found time to finish the change. Maybe you can help by giving feedback on that change?

What kind of feedback? Is it available to use?

**Comment by bjorn** [comment]
> What kind of feedback? Is it available to use?

Yes, you can use it as-is by taking the code from https://github.com/sixthgear/tiled-to-godot-export.
